.gitignore

```diff
@@ -70,6 +70,19 @@ Lib/test/data/*
 /Makefile
 /Makefile.pre
 iOS/Resources/Info.plist
+iOS/testbed/build
+iOS/testbed/Python.xcframework/ios-arm64/bin
+iOS/testbed/Python.xcframework/ios-arm64/include
+iOS/testbed/Python.xcframework/ios-arm64/lib
+iOS/testbed/Python.xcframework/ios-arm64/Python.framework
+iOS/testbed/Python.xcframework/ios-arm64_x86_64-simulator/bin
+iOS/testbed/Python.xcframework/ios-arm64_x86_64-simulator/include
+iOS/testbed/Python.xcframework/ios-arm64_x86_64-simulator/lib
+iOS/testbed/Python.xcframework/ios-arm64_x86_64-simulator/Python.framework
```

Comment:
I think that you can use `*` (something like `ios-*`) to deduplicate these entries for different targets.